packages/core/src/content-delivery/ContentDelivery.ts

```diff
@@ -0,0 +1,380 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import type {
+    Culture,
+    Newsroom,
+    NewsroomTheme,
+    PrezlyClient,
+    Query,
+    TranslatedCategory,
+} from '@prezly/sdk';
+import { ApiError, Category, NewsroomGallery, SortOrder, Stories, Story } from '@prezly/sdk';
+
+export interface Options {
+    formats?: Story.FormatVersion[];
+    pinning?: boolean;
+    cache?: boolean;
+}
+
+export namespace stories {
+    export interface SearchParams {
+        search?: string;
+        category?: Pick<Category, 'id'>;
+        locale?: Pick<Culture, 'code'> | Culture.Code;
+        limit: number;
+        offset?: number;
+        highlighted?: number;
+        query?: Query;
+    }
+
+    export interface IncludeOptions<Include extends keyof Story.ExtraFields> {
+        include?: Include[];
+    }
+}
+
+export namespace allStories {
+    export interface SearchParams {
+        search?: string;
+        category?: Pick<Category, 'id'>;
+        locale?: Pick<Culture, 'code'>;
+    }
+
+    export interface IncludeOptions<Include extends keyof Story.ExtraFields> {
+        include?: Include[];
+    }
+}
+
+export namespace story {
+    export type SearchParams =
+        | { uuid: Story['uuid']; slug?: never }
+        | { uuid?: never; slug: Story['slug'] };
+
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    export interface IncludeOptions<Include extends keyof Story.ExtraFields> {
+        include?: Include[];
+    }
+}
+
+export namespace mediaAlbums {
+    export interface SearchParams {
+        offset?: number;
+        limit?: number;
+        type?: `${NewsroomGallery.Type}`;
+    }
+}
+
+export type Client = ReturnType<typeof createClient>;
+
+export function createClient(
+    prezly: PrezlyClient,
+    newsroomUuid: Newsroom['uuid'],
+    newsroomThemeUuid: NewsroomTheme['id'] | undefined,
+    { formats = [Story.FormatVersion.SLATEJS_V4], pinning = false, cache = false }: Options = {},
```

Comment:
> But considering that this PR will likely produce a major release, it makes sense to remove the option from the exposed methods completely, as mentioned in the deprecation label.

I was thinking that a theme may decide to deliberately disable pinning functionality. 

But it doesn't seem to make a lot of sense. It's harmless and people can just not use it in the UI, if they don't want it. Right?

Reply:
> v6.8.0 enabled pinning by default, so this change should be reflected here as well.

Will do! :+1:

Reply:
> I was thinking that a theme may decide to deliberately disable pinning functionality.

> But it doesn't seem to make a lot of sense. It's harmless and people can just not use it in the UI, if they don't want it. Right?

I had exactly the same thoughts :)
I think it would also make sense to remove the story pinning warnings in the app, along with the `story_pinning` feature awareness flag

Reply:
77afd2367fc3f67fa1f525ab93f7463e23038984